test/ConstructiveSolidGeometry/CSG_coordinates.jl

```diff
@@ -127,3 +127,42 @@ import Unitful: @u_str
         @test_throws ArgumentError CylindricalPoint(1u"m", 2u"rad", 3u"rad")
     end
 end
+
+@testset "CartesianVector" begin
```

Comment:
I see that `CartesianVector` can still have units.
Is this something we want to allow for 0.11? Or to restrict to `Real` and then maybe add back later, when also allowing for units in the coordinates of `CartesianPoint`?

Reply:
But we might need some `to_internal_unit` either when creating the `CartesianVector` or in its operations (but it might not only affect `+` but also other functions, so probably stripping the units when creating a `CartesianVector` is the simplest way to go now.)